ansible/roles/packages/tasks/redhat.yaml

```diff
@@ -1,13 +1,27 @@
 ---
-- name: install common RPMS
+- name: find installed RPMs
```

Comment:
I still don't understand why `yum` here is trying to update but that is the behavior we are seeing.

Reply:
yum skips updating only if the installed rpm version is exactly same as rpm available in the `offline` repo.